src/runtime_src/core/tools/xbmgmt2/flash/xspi.cpp

```diff
@@ -545,12 +545,12 @@ int XSPI_Flasher::xclTestXSpi(int index)
 }
 
 //Method for updating QSPI (expects 1 MCS files)
-int XSPI_Flasher::xclUpgradeFirmware1(std::istream& mcsStream1) {
+int XSPI_Flasher::xclUpgradeFirmware1(std::istream& mcsStream1, std::istream* stripped) {
```

Comment:
Looks strange with one stream being reference and the other a pointer.  If there is a good reason maybe document?